locations/spiders/patel_brothers.py

```diff
@@ -0,0 +1,28 @@
+# # -*- coding: utf-8 -*-
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+
+class PatelBrothersSpider(scrapy.Spider):
+    name = "patel_brothers"
+    item_attributes = {
+        "brand": "Patel Brothers",
+        "brand_wikidata": "Q55641396",
+    }
+    start_urls = ["https://www.patelbros.com/wp-content/plugins/superstorefinder-wp/ssf-wp-xml.php"]
+
+    def parse(self, response):
+        items = response.xpath("//store/item")
+        for item in items:
+            properties = {
+                "lat": item.xpath("latitude//text()").extract_first(),
+                "lon": item.xpath("longitude//text()").extract_first(),
+                "phone": item.xpath("telephone//text()").extract_first(),
+                "addr_full": item.xpath("address//text()").extract_first().replace("<br />", ","),
+                "street_address": item.xpath("address//text()").extract_first().replace("<br />", ","),
```

Comment:
`street_address` is important to include where possible. In this case, it looks like it's the part of `<address>` before the `<br />`. Instead of replacing it with `,`, can you split on it and take the first part?